src/sentry/static/sentry/app/components/events/interfaces/crashHeader/crashTitle.tsx

```diff
@@ -33,7 +33,7 @@ const CrashTitle = ({
         <GuideAnchor target="exception" disabled={hideGuide} position="bottom">
           {title}
         </GuideAnchor>
-        <Tooltip title={t('Toggle stacktrace order')}>
+        <Tooltip title={t('Toggle stack trace order')}>
```

Comment:
Are we making this change app wide?

Reply:
We were told that this is the correct way to write it. I've already seen it written like this in docs so I think I'll prepare a PR that changes it in the entire app.

Reply:
https://github.com/getsentry/sentry/pull/22048